src/sections/Sponsor.jsx

```diff
@@ -139,45 +157,56 @@ const Skip = styled.div`
 `
 
 export default function Sponsor () {
-  const [sponsors, setSponsors] = useState(null)
+  const [sponsors, setSponsors] = useState({})
 
   useEffect(async () => {
     const data = await fireDb.getCollection('HackCamp2023', 'Sponsors')
     if (data) {
-      setSponsors(data)
+      const organizedSponsors = {}
+      data.forEach((sponsor) => {
+        if (!organizedSponsors[sponsor.tier]) {
+          organizedSponsors[sponsor.tier] = []
+        }
+        organizedSponsors[sponsor.tier].push(sponsor)
+      })
+      setSponsors(organizedSponsors)
     }
   }, [])
 
-  return sponsors?.length > 0
+  return Object.keys(sponsors).length > 0
     ? (
     <BgSectionContainer id="sponsors">
       <UFO></UFO>
       <UFOLight/>
-      <StyledTitle>
-        Sponsors
-      </StyledTitle>
       <Sponsors>
-        {sponsors.map((sponsor) => (
-          <SponsorLogo src={sponsor.imgURL}/>
+        <StyledTitle>
+          Sponsors
+        </StyledTitle>
+        {Object.keys(sponsors).sort((a, b) => sponsorTierOrder[a] - sponsorTierOrder[b]).map((key) => (
+          <SponsorTier>
+            {sponsors[key].map((sponsor) => (
+              <SponsorLogo src={sponsor.imgURL}/>
```

Comment:
Would it be possible to link to the link contained in `sponsor.link`?
![image](https://github.com/nwplus/external-templates/assets/62491197/c70b9fcf-e6ec-4fc5-bcc8-2c661c8788f8)

Reply:
yer added